src/actions/channels.js

```diff
@@ -1283,6 +1283,32 @@ export function unfavoriteChannel(channelId) {
     };
 }
 
+export function updateChannelScheme(channelId, schemeId) {
+    return async (dispatch, getState) => {
+        dispatch({type: ChannelTypes.UPDATE_CHANNEL_SCHEME_REQUEST}, getState);
```

Comment:
It's there any reason to not use the `bindClientFunc`?

Reply:
Yes, the reason is that `bindClientFunc` forwards the json response from the server to the reducer, so in a case such as this with a response body of `{"status": "OK"}` one wouldn't have any useful parameters to use in the reducer. One must manually create the parameters for the reducer as done on line 1304.

Reply:
Yes, you are right.

Reply:
Come to think of it one *could* still use `bindClientFunc` and wrap the `clientFunc` parameter in a custom function that ultimately returns the data in the format one wants. That is another option... I'm going to try it and see if it saves keystrokes.